config/example/node.yaml

```diff
@@ -1,159 +1,159 @@
-  dial_timeout: 30s
-  disable_cache: true
-  rpc_endpoint:
+  dial_timeout: 30s  # timeout for side chain NEO RPC client connection
+  disable_cache: true  # do not use TTL cache for side chain GET operations
+  rpc_endpoint:  # side chain NEO RPC endpoints; are shuffled and used one by one until the first success
     - https://rpc1.morph.fs.neo.org:40341
     - https://rpc2.morph.fs.neo.org:40341
-  notification_endpoint:
+  notification_endpoint:  # side chain NEO RPC notification endpoints; are shuffled and used only the first non-error one
     - wss://rpc1.morph.fs.neo.org:40341/ws
     - wss://rpc2.morph.fs.neo.org:40341/ws
   notary:
-    deposit_amount: 100000001
-    deposit_duration: 1001
+    deposit_amount: 100000001  # notary deposit GAS amount
+    deposit_duration: 1001  # notary deposit period in blocks
 
-mainchain:
-  dial_timeout: 30s
-  rpc_endpoint:
+mainchain:  # DEPRECATED section, is not used and not read
+  dial_timeout: 30s  # timeout for main chain NEO RPC client connection
+  rpc_endpoint:  # main chain NEO RPC endpoints; are shuffled and used one by one until the first success
     - https://rpc1.n3.nspcc.ru:30341
     - https://rpc2.n3.nspcc.ru:30341
 
 apiclient:
-  dial_timeout: 15s
+  dial_timeout: 15s  # timeout for NEOFS API client connection
 
 policer:
-  head_timeout: 15s
+  head_timeout: 15s  # timeout for the Policer HEAD remote operation
 
 replicator:
-  put_timeout: 15s
+  put_timeout: 15s  # timeout for the Replicator PUT remote operation
 
 object:
   put:
-    pool_size_remote: 100
-    pool_size_local: 101
+    pool_size_remote: 100  # number of async workers for remote PUT operations
+    pool_size_local: 101  # number of async workers for local PUT operations
 
 storage:
-  shard_num: 2
+  shard_num: 2  # total number of shards
   shard:
     0:
-      refill_metabase: false
+      refill_metabase: false  # sync metabase with blobstore on start, expensive, leave false until complete understanding
 
-      use_write_cache: false
+      use_write_cache: false  # use write-cache
 
       writecache:
-        path: tmp/0/cache
-        mem_size: 2147483648
-        small_size: 16384
-        max_size: 134217728
-        workers_number: 30
-        size_limit: 3221225472
+        path: tmp/0/cache  # write-cache root directory
+        mem_size: 2147483648  # approximate RAM usage limit for "small" objects, bytes
+        small_size: 16384  # size threshold for "small" objects which are cached in key-value DB, not on FS, bytes
+        max_size: 134217728  # size threshold for "big" objects which bypass write-cache and go to the storage directly, bytes
+        workers_number: 30  # number of write-cache flusher threads
+        size_limit: 3221225472  # approximate write-cache total size, bytes
 
       metabase:
-        path: tmp/0/meta
-        perm: 0644
+        path: tmp/0/meta  # metabase path
+        perm: 0644  # permissions for metabase files(directories: +x for current user and group)
```

Comment:
Should this value ever be configurable? How the node behaviour would change if I put `0777`; `0600`? 

The same question for blobstor files permissions.

Reply:
Moreover, I believe default permissions must not be all readable.

Reply:
Example file is for humans, not for testing propose. Moreover it contains configuration notes.
It is perceived as a good starting point and permissions with non zeros at the end looks strange.

Reply:
Moreover, good example configs describe default settings as commented out.